css/components/style.css

```diff
@@ -18,3 +16,112 @@ body {
     font-weight: 300;
     font-style: normal;
 }
+
+#section-1 {
+  height: 670px;
+  position: relative;
+}
+
+.mw-background {
+  background: url('/assets/images/intro.jpg') no-repeat center center fixed;
+    background-size: 100%;
+    width:100%;
+    height: 100%;
+    opacity: 0.6;
+}
+
+.mw-header {
+  position: absolute;
+    top: 50%;
+    left: 15%;
+    font-size: 100px;
+}
+
+.mw-container-1 {
+  padding: 150px 200px 0;
+  height: 500px;
+  background-color: white;
+}
+
+.mw-container-1 > .mw-container-left {
+  height: 250px;
+  width: 50%;
+  display: inline-block;
+}
+
+.mw-container-1 > .mw-container-left > img {
+  width: 100%;
+}
+
+.mw-container-1 > .mw-container-right {
+  float: right;
+  height: 250px;
+  width: 50%;
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+.mw-container-1 > .mw-container-right > span:nth-child(1) {
```

Comment:
daca folosesti **div**-uri in loc de span-uri nu mai esti nevoit sa pui `display: block`